tests/percy/multilangphotographer.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * @typedef {import('./pagenavigator.js')} PageNavigator
+ * @typedef {import('./Camera.js')} Camera
+ */
+
+/**
+ * Responsible for determining which snapshots to take for specific locale
+ */
+class MultilangPhotographer {
+  /**
+   * @param {PageNavigator} pageNavigator
+   * @param {Camera} camera
+   * @param {locale} locale locale set for the site when taking snapshot
+   */
+  constructor(pageNavigator, camera, locale) {
+    this._pageNavigator = pageNavigator;
+    this._camera = camera;
+    this._locale = locale;
+
+    /**
+     * Custom queries for specific locale
+     * 
+     * @type {Object<string, string>}
+     */
+    this._queries = {};
+
+    /**
+     * Locale tag to add to snapshot name
+     * 
+     * @type {string}
+     */
+    this.localeSnapshotTag = '';
+
+    this.setLocale(locale);
+  }
+
+  /**
+   * Sets locale for the page navigation and snapshot tag
+   * 
+   * @param {string} locale locale of the site
+   */
+  setLocale(locale) {
+    this._locale = locale;
+    this.localeSnapshotTag = this._locale === 'en'? '' : this._locale + '--';
+    this._setLocaleQueries();
+  }
+
+  /**
+   * Sets custom queries based on locale
+   */
+  _setLocaleQueries() {
+    switch (this._locale) {
+      case 'es':
+        this._queries = {
+          faq: '¿Qué pasa si olvidé mi contraseña?',
+          job: 'trabajo',
+          menu_item: 'rollo',
+          field_direct_answers: 'descripción de bryan reed'
+        }
+        break;
+      default:
+        this._queries = {
+          faq: 'what if i forgot my password?',
+          job: 'job',
+          menu_item: 'roll',
+          field_direct_answers: 'bryan reed description'
+        }
+    }
+  }
+
+  async captureSnapshots() {
+    await this._captureUniversalSearch();
+    await this._captureVerticalSearch();
+    await this._captureVerticalGridSearch();
+    await this._captureVerticalMapSearch();
+    await this._captureVerticalFullPageMapSearch();
+    await this._captureDirectAnswers();
+  }
+
+  async _captureUniversalSearch () {
+    await this._pageNavigator.gotoUniversalPage();
+    await this._camera.snapshot(this.localeSnapshotTag + 'universal-search');
```

Comment:
Rather than a localeSnapshotTag, what do you think of having a locale property in camera, and if it is set, the camera will prepend the locale to the snapshot?